### PR TITLE
staging-node16 -> redesign-2021

### PR DESCRIPTION
**Pull request by alexey-petrov**

### CODE REVIEW

File: src/components/molecules/SpaceEditForm/SpaceEditForm.tsx [hunk 1]
```diff
@@ -426,21 +426,21 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
...
                     label="Show reactions"
                     variant="toggler"
                     register={register}
+                    name="showReactions"
                   />
                   <AdminCheckbox
                     variant="flip-switch"
-                    name="isReactionsMuted"
                     register={register}
+                    name={"isReactionsMuted"}
```
Comment (goran-peoski-work) [inline src/components/molecules/SpaceEditForm/SpaceEditForm.tsx:443]:
```suggestion
                    name="isReactionsMuted"
```

**Review by goran-peoski-work — APPROVED** [review]
LGTM

two typos and a console.log forgotten

File: src/components/molecules/SpaceEditForm/SpaceEditForm.tsx [hunk 2]
```diff
@@ -277,17 +277,17 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
...
+                name="subtitle"
                 errors={errors}
               />
             </AdminSection>
             <AdminSection title="Description" withLabel>
               <AdminTextarea
-                name="description"
                 placeholder={`Let your guests know what they’ll find when they join your space. Keep it short & sweet, around 2-3 sentences maximum. Be sure to indicate any expectations for their participation.`}
                 register={register}
+                name={"description"}
```
[the remark this answers is not in the document]
Comment (alexey-petrov) [reply src/components/molecules/SpaceEditForm/SpaceEditForm.tsx]: [ebce8d3](https://github.com/sparkletown/sparkle/pull/2862/commits/ebce8d3d7f91c25590187944e6b2344b4272c944)

File: src/components/molecules/SpaceEditForm/SpaceEditForm.tsx (hunk 1 above)
Comment (alexey-petrov) [reply src/components/molecules/SpaceEditForm/SpaceEditForm.tsx]: [ebce8d3](https://github.com/sparkletown/sparkle/pull/2862/commits/ebce8d3d7f91c25590187944e6b2344b4272c944)

File: src/pages/Admin/Venue/VenueAdminPage.tsx [hunk 3]
```diff
@@ -21,20 +20,26 @@ import { AnnouncementOptions } from "./AnnouncementOptions";
...
-  const { space, spaceId, isLoaded } = useWorldAndSpaceBySlug(
-    worldSlug,
-    spaceSlug
+  const { profile, auth: user, userId } = useUserNG();
+  const { space, spaceId, isLoaded } = useWorldAndSpaceByParams();
+  const { currentVenue, parentVenue } = useRelatedVenues({
+    currentVenueId: spaceId,
+  });
+
+  console.log(VenueAdminPage.name, userId, user, profile);
```
[the remark this answers is not in the document]
Comment (alexey-petrov) [reply src/pages/Admin/Venue/VenueAdminPage.tsx]: [ebce8d3](https://github.com/sparkletown/sparkle/pull/2862/commits/ebce8d3d7f91c25590187944e6b2344b4272c944)

**Pull request merged**